invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/theme.js

```diff
@@ -37,11 +37,19 @@ $tabElement.on("keydown", function (event) {
   }
 });
 
-
+const affiliationsButton = $('.title .affiliations-button');
 $(".ui.accordion.affiliations").accordion({
```

Comment:
https://github.com/inveniosoftware/invenio-requests/pull/221 I think this global .ui.accordion class implementation covers also this element, no?

Reply:
No, this accordion title has a button as a child, which is the trigger.

Reply:
so maybe it would make sense to unify them ? to have the same implementation? is there any reason why we need one being triggered by title and the other by button?

Reply:
We don't want it to be triggered by the entire title if there is a button, because when it's spanning the full width of the container, that entire field will be clickable, not only the button.

Reply:
The selector is specified in the init function, and we can only have one init function per accordion, otherwise they will override each other.